sql/maintenance_utils.sql

```diff
@@ -42,6 +42,17 @@ CREATE OR REPLACE FUNCTION @extschema@.decompress_chunk(
     if_compressed BOOLEAN = false
 ) RETURNS REGCLASS AS '@MODULE_PATHNAME@', 'ts_decompress_chunk' LANGUAGE C STRICT VOLATILE;
 
+-- experimental at first
+CREATE OR REPLACE FUNCTION @extschema@.recompress_chunk_experimental(
+    uncompressed_chunk REGCLASS,
+    if_compressed BOOLEAN = false
+) RETURNS REGCLASS AS '@MODULE_PATHNAME@', 'ts_recompress_chunk_experimental' LANGUAGE C STRICT VOLATILE;
```

Comment:
I think we want to tune this a little, i dont think we wanna call it experimental and we probably dont wanna put it in public schema

Reply:
Renamed it to `recompress_chunk_segmentwise` and moved to the `_timescaledb_internal` schema